mutalyzer/variant.py

```diff
@@ -0,0 +1,270 @@
+"""
+"""
+
+from __future__ import unicode_literals
+
+from Bio.SeqUtils import seq3
+
+from extractor import extractor
+
+from mutalyzer import models
+
+weights = {
+    "subst": extractor.WEIGHT_SUBSTITUTION,
+    "del": extractor.WEIGHT_DELETION,
+    "ins": extractor.WEIGHT_INSERTION,
+    "dup": extractor.WEIGHT_INSERTION,
+    "inv": extractor.WEIGHT_INVERSION,
+    "delins": extractor.WEIGHT_DELETION_INSERTION
+}
+
+class HGVSList(list):
+    """
+    Container for a list of sequences or variants.
+    """
+    def __unicode__(self):
+        if len(self) > 1:
+            return "[{}]".format(';'.join(map(unicode, self)))
+        return unicode(self[0])
+    #__unicode__
+
+    def weight(self):
+        weight = sum(map(lambda x: x.weight(), self))
+
+        if len(self) > 1:
+            return weight + (len(self) + 1) * extractor.WEIGHT_SEPARATOR
+        return weight
+    #weight
+#HGVSList
+
+class Allele(HGVSList):
```

Comment:
Could use a brief description.

This class name is also used in `models`, would be better to prevent this.